### DISCUSSION TITLE
Feature request: More particle picking integration - Topaz and crYOLO

### DISCUSSION THREAD

**Discussion by alexjnoble** [nextpyp/.github#18] [Ideas]
Hello,

It would be useful to have more particle picking options, such as Topaz and crYOLO. Topaz is distributed under GPLv3, so it should be integratable without issue. CrYOLO has its own license, which may be too restrictive.

Thanks,
-Alex

**Comment by abartesaghi** [comment]
Leaving the issue of licenses aside, it should not be difficult to integrate these into nextPYP. We will look into how best to do this while complying with all the licenses.

**Reply by abartesaghi** [reply]
Support for 2D particle picking using Topaz is available starting with version [0.6.0](https://nextpyp.app/files/pyp/0.6.0/docs/index.html).
